build/images/Dockerfile.build.windows

```diff
@@ -14,6 +14,7 @@
 
 ARG GO_VERSION
 ARG OVS_VERSION
+ARG DOCKER_REGISTRY="docker.io"
```

Comment:
can you point me to the place where we set this build arg?

Reply:
I updated related code in build/images/build-windows.sh.